warehouse/templates/includes/flash-messages.html

```diff
@@ -16,7 +16,7 @@
 <div class="notification-bar notification-bar--warning" data-controller="notification" data-target="notification.notification">
   <div class="site-container">
     <span class="notification-bar__message">{{ message }}</span>
-    <span data-target="notification.dismissButton" data-action="click->notification#dismiss" class="notification-bar__dismiss">×</span>
```

Comment:
Could we please make this `<span>` a `<button>` and use a font awesome icon? Also, we could probably add some aria in there:

```
<button data-target="notification.dismissButton" data-action="click->notification#dismiss" class="notification-bar__dismiss" aria-label="close">
  <i class="fa fa-times"></a>
</button>
```